src/parquet/thrift/serializer-test.cc

```diff
@@ -0,0 +1,89 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include <cstdlib>
+#include <iostream>
+#include <sstream>
+#include <string>
+
+#include <gtest/gtest.h>
+
+#include "parquet/thrift/parquet_types.h"
+#include "parquet/thrift/util.h"
+
+using std::string;
+
+namespace parquet_cpp {
+
+class TestThrift : public ::testing::Test {
+ public:
+ void initStats(size_t stat_size, parquet::Statistics &stat) {
+  std::vector<char> stat_buffer;
+  stat_buffer.resize(stat_size);
+  for(int i = 0; i < stat_size; i++){
+    (reinterpret_cast<uint8_t*>(stat_buffer.data()))[i] = i%255;
+  }
+  stat.__set_max(std::string(stat_buffer.data(), stat_size));
+ }
+ void initDataPage(parquet::DataPageHeader &data_page, parquet::Statistics &stat) {
+  data_page.encoding = parquet::Encoding::PLAIN;
+  data_page.definition_level_encoding = parquet::Encoding::RLE; 
+  data_page.repetition_level_encoding = parquet::Encoding::RLE;
+  data_page.__set_statistics(stat);
+ }
+ void initPageHeader(parquet::PageHeader &page_header, 
+     parquet::DataPageHeader &data_page) {
+  page_header.__set_data_page_header(data_page);
+ }
+};
+
+TEST_F(TestThrift, TestSerializerDeserializer) {
```

Comment:
BTW thanks for adding this unit test. It will also be useful to have reusable tools for microbenchmarking